diff --git a/mingw-w64-headers/configure b/mingw-w64-headers/configure
index 0af98e16..adbaa47a 100755
--- a/mingw-w64-headers/configure
+++ b/mingw-w64-headers/configure
@@ -1345,7 +1345,7 @@ Optional Packages:
   --with-widl=PATH        use widl to update idl-based headers. Optionally
                           search for a compatible widl in PATH
   --with-default-win32-winnt=VER
-                          Default value of _WIN32_WINNT (default: 0x502)
+                          Default value of _WIN32_WINNT (default: 0x600)
   --with-default-msvcrt=LIB
                           Default msvcrt to target (default: msvcrt)
 
@@ -2826,7 +2826,7 @@ $as_echo_n "checking default _WIN32_WINNT version... " >&6; }
 if test "${with_default_win32_winnt+set}" = set; then :
   withval=$with_default_win32_winnt;
 else
-  with_default_win32_winnt=0x502
+  with_default_win32_winnt=0x600
 fi
 
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: $with_default_win32_winnt" >&5
diff --git a/mingw-w64-headers/configure.ac b/mingw-w64-headers/configure.ac
index dda8a629..5a50e43d 100644
--- a/mingw-w64-headers/configure.ac
+++ b/mingw-w64-headers/configure.ac
@@ -155,9 +155,9 @@ AC_SUBST([MINGW_HAS_SECURE_API])
 AC_MSG_CHECKING([default _WIN32_WINNT version])
 AC_ARG_WITH([default-win32-winnt],
   [AS_HELP_STRING([--with-default-win32-winnt=VER],
-    [Default value of _WIN32_WINNT (default: 0x502)])],
+    [Default value of _WIN32_WINNT (default: 0x600)])],
   [],
-  [with_default_win32_winnt=0x502])
+  [with_default_win32_winnt=0x600])
 AC_MSG_RESULT([$with_default_win32_winnt])
 AS_VAR_SET([DEFAULT_WIN32_WINNT],[$with_default_win32_winnt])
 AC_SUBST([DEFAULT_WIN32_WINNT])
